src/extractLevels.js

```diff
@@ -0,0 +1,170 @@
+import fs from 'fs'
+import IfcModel from './IfcModel.js'
+
+
+let ifcFilename = 'IFCFile'
+const IFCSpacesELementsArrAll = []
+const USAGE = `Usage: node extractLevel.js <file.ifc>
```

Comment:
should be able to copy-paste this to execute from proj root dir.
```
node src/extractLevels.js
```

Reply:
I'll change the output messages to a more meaningful argument, and I'll run some tests on ifc files other than haus.ifc. Can you elaborate on the *.test.js file? Currently, I am using the IFC.js viewer to check the results.